src/sass-inject.js

```diff
@@ -23,10 +18,9 @@ const importSass = new Promise((resolve, reject) => {
 // intercept file loading requests (@import directive) from libsass
 importSass.then(sass => {
   sass.importer((request, done) => {
-    const { current } = request;
     // Currently only supporting scss imports due to
     // https://github.com/sass/libsass/issues/1695
-    const importUrl = url.resolve(urlBase, `${current}.scss`);
+    const importUrl = url.resolve('', `${request.resolved}.scss`);
```

Comment:
`request.resolved` is a path with /sass/ as it's root.  This mandates all scss files be within the /sass/ folder in the root of the project.  It seems like this could be changed in sass.worker.js somehow.